documentation/docs/scenarios/ui5-sap-cp/Readme.md

```diff
@@ -55,7 +55,7 @@ This is a basic configuration example, which is also located in the sources of t
 steps:
   mtaBuild:
     buildTarget: 'NEO'
-    mtaJarLocation: '/opt/sap/mta.jar'
+    platform: 'neo'
```

Comment:
Are we keeping `buildTarget` and `platform` because of backwards-compatibility with the legacy build tool?

Reply:
No, you have to explicitly say where you want to go. In this case platform is per default "cf" and this clashes with the buildTarget.